VPNShield/VPNShield.cs

```diff
@@ -63,7 +67,9 @@ public override void Register()
             this.AddCommand("vs_disable", new DisableCommand(this));
             this.AddCommand("vs_whitelist", new WhitelistCommand(this));
 			this.AddConfig(new ConfigSetting("vs_global", true, true, "Whether or not to use the global config directory, default is true"));
-        }
+			this.AddConfig(new ConfigSetting("vs_no_purchases_message", "This server does not allow new Steam accounts, you have to buy something on Steam before playing.", true, "The message to display to players who are kicked for having a steam account without a purchased game."));
+			this.AddConfig(new ConfigSetting("vs_non_setup_message", "This server does not allow non setup Steam accounts, you have to setup your Steam profile before playing.", true, "The message to display to players who are kicked for having a Steam account that is not yet setup."));
```

Comment:
These should not be vanilla config options, they should be part of the plugin config.